opentelemetry-sdk/src/opentelemetry/sdk/_metrics/view.py

```diff
@@ -0,0 +1,172 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=protected-access
+
+from dataclasses import dataclass
+from logging import getLogger
+from re import match
+from typing import Dict
+
+from opentelemetry._metrics.instrument import Instrument
+from opentelemetry.sdk._metrics.aggregation import Aggregation
+
+_logger = getLogger(__name__)
+
+
+@dataclass
```

Comment:
I don't think this is actually doing anything. I guess I take back what I said about dataclasses since there is a bunch of logic 👍 sorry!

Reply:
I am ok with leaving this in, it provides a representation of the view, having logic or not does no affect this.

Reply:
It actually doesn't know about any of the fields unless you declare them as static attributes of the dataclass. So as is, `str(view)` won't be too useful

Reply:
All right, removing decorator